pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -79,12 +79,12 @@ func getNetworksAndCniNetworks(vmi *v1.VirtualMachineInstance) (map[string]*v1.N
 	return networks, cniNetworks
 }
 
-func invokeNetworkInterfaceFactory(networks map[string]*v1.Network, ifaceName string) (NetworkInterface, error) {
+func invokepodNICFactory(networks map[string]*v1.Network, ifaceName string) (podNIC, error) {
```

Comment:
Even though `podNic` is private, in the method name it is not relevant. So I would use - `invokePodNICFactory`.